Src/BootCamp.Chapter/ShuffleDemo.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace BootCamp.Chapter
+{
+    internal static class ShuffleDemo
+    {
+        internal static void Demo()
+        {
+            var shuffledList = ShuffleColllectionExtension(new List<int> { 1, 2, 3, 4, 5 });
+            Console.Write("The shuffled List looks like this: ");
+            foreach (var item in shuffledList)
+            {
+                Console.Write($"{ item} ");
+            }
+            Console.WriteLine(Environment.NewLine); 
+        }
+
+        private static IEnumerable<T> ShuffleColllectionExtension<T>(this IEnumerable<T> list)
```

Comment:
Should be public and like that demonstrate that you can call this as an extension.

Reply:
Demonstrate that you understand what extension methods are for. Here you demonstrate that you misunderstand the difference between a static class and extension methods.

Reply:
oke, I have to think what you want here exactly. Sometimes you are very cryptic and I do then not see what you mean